runtime/connectors/gzip.go

```diff
@@ -0,0 +1,36 @@
+package connectors
+
+import (
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+func extractGzipFile(fileName string) (string, error) {
```

Comment:
This function doesn't delete the temporary unzipped file after it has been used. Maybe there's a nicer way to design the code, but a simple solution is to just have the caller defer delete it

Reply:
Have moved the unzip responsibility to duckdb. Better to leverage its code for now. We can revisit it for druid